src/libraries/Microsoft.Extensions.Hosting.Abstractions/src/BackgroundService.cs

```diff
@@ -28,6 +28,7 @@ public abstract class BackgroundService : IHostedService, IDisposable
         /// the lifetime of the long running operation(s) being performed.
         /// </summary>
         /// <param name="stoppingToken">Triggered when <see cref="IHostedService.StopAsync(CancellationToken)"/> is called.</param>
+        /// See <see href="https://docs.microsoft.com/dotnet/core/extensions/workers">Worker Services in .NET</see> for implementation guidelines.
```

Comment:
```suggestion
        /// <related type="Article" href="/dotnet/core/extensions/workers">Worker Services in .NET</related>
```

To follow the example @gewarren linked to: https://github.com/dotnet/runtime/blob/b2ab353c0ed8eb9d394570a7cde8c4ed03c771e5/src/libraries/System.Private.CoreLib/src/System/Numerics/Vector2.cs#L691

Reply:
But that is a bare reference, with no information why it is there.  I would prefer to have it in context.  Please confirm.

Reply:
If you decide to keep the `<see>` tag, you'll need to put the entire text within the `summary` or `remarks` tags.

Reply:
The docs understand this format, and it makes it look like this:

https://docs.microsoft.com/en-us/dotnet/api/system.numerics.vector2.tostring?view=net-6.0#System_Numerics_Vector2_ToString_System_String_

![image](https://user-images.githubusercontent.com/8291187/148264207-5a5f6be9-9f63-41f4-a84d-99a421d49ef6.png)

Reply:
Yes, and that is for non-essential references, like when you can safely assume the reader already knows these things.  This is not the case here.